package.json

```diff
@@ -930,6 +939,11 @@
         "view": "openshiftComponentsView",
         "contents": "You have no OpenShift Components in your workspace yet.\n[New Component](command:openshift.component.createFromLocal)",
         "enablement": "openshift.component.explorer.init"
+      },
+      {
+        "view": "openshiftComponentTypesView",
+        "contents": "Default Devfile Regisitries are Here.\n[Open Default Devfile Registry](command:openshift.component.open.devfileRegistry)",
+        "enablement": "openshift.component.explorer.init"
```

Comment:
This enablement condition is not correct. It should be specific to Component Types View, not Components view (which shows components in workspace).